create_MyString_file.py

```diff
@@ -0,0 +1,109 @@
+class MyString:
+    def __init__(self,s):
+        self.string = s
+        self.length = MyString.len(self)
+
+    def __str__(self):
+        return self.string
+
+    def len(self):
+        count = 0
+        for char in self.string:
+            count += 1
+        return count
+
+    def add_string(self, other):
+        return '{}{}'.format(self.string,other)
+
+    def substring(self,start,end):
+        if start > end:
+            return None
+
+        substr = ''
+        for i in range(start,end):
+            if i >= self.length:
+                break
+            substr = MyString(substr)
+            substr = substr.add_string(self.string[i])
+        return substr
+
+    def insert_string(self,pos,other):
+        if pos > self.length:
+            return "Position is exceed the length of string!"
+
+        st_part = MyString.substring(self,0,pos)
+        last_part = MyString.substring(self,pos,self.length)
+        # ins_str =  st_part + other + last_part
+        st_part = MyString(st_part)
+        ins_str = MyString(st_part.add_string(other)).add_string(last_part)
+        return ins_str
+
+    def delete_string(self,start,end):
+        if start > end:
+            return self.string
+
+        st_part = MyString.substring(self,0,start)
+        last_part = MyString.substring(self,end,self.length)
+        if last_part is None:
+            return st_part
+
+        # del_str = st_part + last_part
+        st_part = MyString(st_part)
+        del_str = st_part.add_string(last_part)
+        return del_str
+
+    def replace_string(self,start,end,other):
+        if start > end:
+            return self.string
+
+        steps = end - start
+        st_part = MyString.substring(self,0,start)
+        last_part = MyString.substring(self,end,self.length)
+        st_part = MyString(st_part)
+
+        added_part = ''
+        for i in range(0,steps):
```

Comment:
dont need for loop here, just get substring of `other` and concat start + substring of `other` + end